src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1467,6 +1474,9 @@ type Itinerary {
     """How far the user has to walk, in meters."""
     walkDistance: Float
 
+    """Emissions of the trip on this itinerary."""
+    emissions: Emissions
```

Comment:
I think the field should be called `emissionsPerPerson`. Also, the comment could be just `Emissions of this itinerary per traveler.` as there can be multiple "trips" within the itinerary and car legs also have emissions and we don't typically call them trips in the terminology here.

Reply:
Ah yes, thanks! The terminology is quite hazy for me still.